docs/migration/migrate_8_0.asciidoc

```diff
@@ -80,4 +80,13 @@ specified explicitly.
 
 *Impact:* Any workflow that involved manually clearing generated bundles will have to be updated with the new path.
 
+[float]
+=== Configurations starting with `xpack.telemetry.` are no longer valid
+
+*Details:*
+The `xpack.` prefix has been removed for all telemetry configurations.
+
+*Impact:*
+For any configurations beginning with `xpack.telemetry.`, remove the `xpack.` prefix. For example, use <<telemetry.enabled, `telemetry.enabled`>> instead.
```

Comment:
Because this content gets pulled into the Stack docs, the link needs to be formatted as follows:

{kibana-ref}/monitoring-settings-kb.html#monitoring-general-settings[telemetry.enabled]

Note:  This link goes to the section heading because I was unable to get the link to telemetry.enabled to work.  Also, linking to the section heading gives the reader more context.

Reply:
@afharo i believe we need to address this one before merging.

Reply:
yup! It's even failing the build. I'll work on it today...

I was wondering why we have that config parameter in the monitoring file instead of a specific doc file for telemetry where we can explain all the configuration properties.